package.json

```diff
@@ -10,8 +10,9 @@
   ],
   "license": "MIT",
   "scripts": {
+    "generate-schema": "node ./dslToJson.js > graphql_schema.json",
     "postinstall": "node ./copyPlatformBinaryInPlace.js",
-    "generate-schema": "node ./dslToJson.js > graphql_schema.json"
+    "postpublish": "dune-release"
```

Comment:
I think we should probably do this manually for now.

Reply:
removed :ok_hand: